### PR TITLE
Pull Requests for the possibility of creating a symfony recipe

### PR DESCRIPTION
**Pull request by aLkRicha**
I am targeting this branch, because it's latest branch.

Created for this [PR](https://github.com/symfony/recipes-contrib/pull/235)

## Changelog

```markdown
### Added
- Added `symfony/twig-bundle` dependency
```

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -23,7 +23,7 @@
         "sonata-project/core-bundle": "^3.4",
         "symfony/form": "^2.8 || ^3.2 || ^4.0",
         "symfony/http-kernel": "^2.8 || ^3.2 || ^4.0",
-        "twig/twig": "^1.34 || ^2.0"
```
Comment (core23) [inline composer.json]: Why did you remove twig? We still need twig for our templates

Comment (aLkRicha) [reply composer.json]: @core23 https://packagist.org/packages/symfony/twig-bundle requires twig

Comment (core23) [reply composer.json]: That's right, but we can require other version of twig. Maybe a higher version than symfony does

Comment (aLkRicha) [reply composer.json]: @core23 yes, some builds fail, if there is lowest version of twig 👎

**Comment by aLkRicha** [comment]
@core23 all 🍏 )

**Comment by aLkRicha** [comment]
@jordisala1991 done.

**Comment by core23** [comment]
Thanks @aLkRicha

**Comment by aLkRicha** [comment]
@core23 is there way to release this asap?

**Comment by core23** [comment]
We just got a release today, but I can ask

**Comment by aLkRicha** [comment]
@core23 thanks 👍

**Comment by Soullivaneuh** [comment]
> Added `symfony/twig-bundle` dependency

This is not a valid changelog entry. Add a dependency, yes, but for what? See my update.

**Comment by jordisala1991** [comment]
Thank you @Soullivaneuh
